hierarchy/src/commonMain/kotlin/treemap/SliceAndDice.kt

```diff
@@ -0,0 +1,41 @@
+package com.juul.krayon.hierarchy.treemap
+
+import com.juul.krayon.hierarchy.Node
+import com.juul.krayon.hierarchy.depth
+
+public object SliceAndDice : TileMethod {
```

Comment:
Is this name pulled from somewhere else? Gets a 👍

Reply:
Shockingly, this name actually comes from academic literature. https://www.cs.umd.edu/~ben/papers/Shneiderman2001Ordered.pdf

Granted, I pulled it from D3. But that's where they got it.